source/sixtrack_input.f90

```diff
@@ -1836,6 +1839,101 @@ subroutine sixin_parseInputLineMULT(inLine, iLine, iErr)
 
 end subroutine sixin_parseInputLineMULT
 
+
+subroutine sixin_parseInputLineRF_MULT(inLine, iLine, iErr)
```

Comment:
Please call the routine `sixin_parseInputLineRFMU` so it matches the block name. This is how the others are implemented.